deployment/quick-start/services-configuration.yaml.template

```diff
@@ -41,6 +41,10 @@ cluster:
     # Must be lower case, e.g., regsecret.
     secret-name: pai-secret
 
+  api-server:
```

Comment:
Don't put this configuration here. It's not a service.

Reply:
this is flag will be used by prometheus / watchdog / webportal. Do you have suggestions it is better to put this config to which location and file?

Reply:
kubernetes-configuraiton

Reply:
Hi @ydye, after refactor, this config move to kubernetes config .  make it as user configure according to their needs